src/content/content.js

```
@@ -1,79 +1,55 @@
 
 function logDebug(message, ...args) {
     if (CONTENT_DEBUG) {
         console.log("[CONTENT_DEBUG]", message, ...args);
     }
 }
 
-/* STUB | PRICING + UTIL CONTENT SCRIPT */
-
-if(typeof commentHeader === "undefined"){
+// Initialize variables
+if (typeof commentHeader === "undefined") {
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)